msmarco-passage-ranking/track.py

```diff
@@ -80,5 +184,128 @@ def params(self):
         }
 
 
+# For each query this will generate the weighted terms query, a pruned version and a rescored pruned version of the same query.
```

Comment:
Looks like the generation logic is in `WeightedTermsRecallRunner` - consider moving the comment.